lib/tasks/helpers/cfe_payload_recorder.rb

```diff
@@ -0,0 +1,40 @@
+class CfePayloadRecorder
+  def self.call(application_ref)
+    new(application_ref).call
+  end
+
+  def initialize(application_ref)
+    @legal_aid_application = LegalAidApplication.find_by(application_ref: application_ref)
+    raise ArgumentError, 'No such application' if @legal_aid_application.nil?
+
+    @recording = []
+  end
+
+  def call
+    submission = @legal_aid_application.most_recent_cfe_submission
+    submission.submission_histories.each do |history|
+      record_history(history)
+    end
+    puts '#'
+    puts "# CFE API calls for applicaiton #{@legal_aid_application.application_ref}"
+    puts '# Copy and paste this output in to tmp/api_replay.yml in the CFE repo, and '
```

Comment:
Is there a way to make this BOLD so that we don't miss the fact that it says " `in the CFE repo` "

Reply:
I don't think so - it is just output at the top of the YAML on the screen.  I guess we can make it in red. Let me try that.